<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateStreamDialog</class>
 <widget class="QWidget" name="AddBranchDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <layout class="QFormLayout" name="search_form">

       <item row="0" column="0">
        <widget class="QLabel" name="name_label">
         <property name="text">
          <string>Branch Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="name_label">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>

       <item row="0" column="1">
        <widget class="QLineEdit" name="name_field"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="description_field"/>
       </item>

      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="dialog_button_box">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>

    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
